pom.xml

```diff
@@ -0,0 +1,96 @@
+
+  <modelVersion>4.0.0</modelVersion>
+  
+  <groupId>org.dataconservancy.pass</groupId>
+  <artifactId>pass-client</artifactId>
+  <packaging>pom</packaging>
+  <version>0.0.1-SNAPSHOT</version>
+  
+  <name>PASS Client Tool</name>
+  
+  <modules>
+    <module>pass-model</module>
+    <module>pass-json-adapter</module>
+    <module>pass-fedora-client</module>
+    <module>pass-client-api</module>
+    <module>pass-model-ext-nihms</module>
+    <module>pass-client-util</module>
+  </modules>
+  
+  <properties>
+    <maven-compiler-plugin.version>3.6.1</maven-compiler-plugin.version>
+    <maven-surefire-plugin.version>2.20.1</maven-surefire-plugin.version>
+    <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <joda-time.version>2.9.9</joda-time.version>
+    <org-json.version>20180130</org-json.version>
+    <junit.version>4.12</junit.version>
+    <jackson.version>2.9.4</jackson.version>
+    <fcrepo-java-client.version>0.3.0</fcrepo-java-client.version>
+    <logback.version>1.2.3</logback.version>
+    <slf4j.version>1.7.25</slf4j.version>
+  </properties>
+
+
+  <build>    
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>${maven-compiler-plugin.version}</version>
+          <configuration>
+            <source>1.8</source>
+            <target>1.8</target>
+          </configuration>
+        </plugin>
+
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-source-plugin</artifactId>
+          <version>${maven-source-plugin.version}</version>
+          <executions>
+            <execution>
+              <goals>
+                <goal>jar</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+    
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <version>${maven-surefire-plugin.version}</version>
+        </plugin>
+        
+      </plugins>
+    </pluginManagement>
+  </build>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+        <version>${slf4j.version}</version>
+      </dependency>
+      
+      <dependency>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
+        <version>${junit.version}</version>
+        <scope>test</scope>
+      </dependency>
+      
+      <dependency>
+        <groupId>ch.qos.logback</groupId>
+        <artifactId>logback-classic</artifactId>
+        <version>${logback.version}</version>
+        <scope>test</scope>
+      </dependency>
```

Comment:
+1 to `dependencyManagement`!  I think the following are still missing:

* `org.fcrepo.client:fcrepo-java-client`
* `com.fasterxml.jackson.core:jackson-databind`
* `org.json:json`
* `joda-time:joda-time`

Then the `<version>` element can be dropped from these dependencies in the module poms, and the parent pom's `<dependencyManagement>/<dependencies>/<dependency>` contains the `<version>`